server/channels/app/platform/web_hub.go

```diff
@@ -360,6 +378,36 @@ func (ps *PlatformService) WebConnCountForUser(userID string) int {
 	return 0
 }
 
+// GetActiveUserIDsForChannel returns user IDs with active WebSocket connections in a channel,
+// excluding the specified user ID (typically the user in session).
+// This iterates across all hubs to find users with active connections.
+// It leverages the byChannelID index when EnableWebHubChannelIteration is enabled for optimal performance.
+func (ps *PlatformService) GetActiveUserIDsForChannel(channelID, excludeUserID string) []string {
+	// Guard against empty channel ID or no hubs
+	if channelID == "" || len(ps.hubs) == 0 {
+		return []string{}
+	}
+
+	// Use map[string]struct{} for memory efficiency (follows model.StringSet pattern)
+	userIDMap := make(map[string]struct{}, 256)
+
+	// Iterate through all hubs to collect active users
+	for _, hub := range ps.hubs {
+		userIDs := hub.GetActiveUserIDsForChannel(channelID, excludeUserID)
+		for _, userID := range userIDs {
+			userIDMap[userID] = struct{}{}
+		}
+	}
+
+	// Convert map to slice
+	userIDs := make([]string, 0, len(userIDMap))
+	for userID := range userIDMap {
+		userIDs = append(userIDs, userID)
+	}
+
+	return userIDs
```

Comment:
Have you considered using `https://pkg.go.dev/maps#Keys`? Not sure if the implementation is more optimized or not, so we would need to check whether it is worth it.